readme.md

```diff
@@ -73,7 +74,31 @@ you can confirm this worked correctly by visiting http://localhost:4000/v1/searc
 
 > note: this guide only covers importing the admin areas (like cities, countries etc.)
 
-#### Downloading the Data
+#### Downloading the Data (easy way)
```

Comment:
Did you mean to put "Downloading the Data (easy way)" in its own section? It's under "Importing Whosonfirst" right now but this talks about OSM, OA, and TIGER data.